Gemfile

```diff
@@ -2,4 +2,8 @@
 # If you modify this file manually all comments and formatting will be lost.
 
 source "https://rubygems.org"
-gemspec :name => "logstash-core"
+gem "logstash-core", "2.0.0.dev", :path => "."
+gem "ci_reporter", "1.9.3", :group => :development
```

Comment:
I don't get the context of this change, what is the criteria to have dependencies on the gemspec or in the Gemfile? why only the testing dependencies are here? Do we keep only in the gemspec dependencies that are runtime? 

is there any problem on grouping all the testing dependencies in the Gemfile like:

```
group :developement do 
 blablabla
end
```